.github/workflows/claude-master.yml

```
@@ -1,135 +1,35 @@
-    workflows: ["RN Unit CI"]
-    types: [completed]
-    branches: [main]
-  issue_comment:
-    types: [created]
   workflow_dispatch:
     inputs:
       action_type:
         description: '実行アクション'
         required: true
-        default: 'auto_fix'
+        default: 'quality_check' # デフォルトを quality_check に変更
```

actionlint (1.7.7)

default value "quality_check" of "action_type" input is not included in its options "\"run_claude_autofix\", \"run_quality_check\""

(events)